components/resources/wallet_strings.grdp

```diff
@@ -1145,6 +1145,11 @@
   <message name="IDS_BRAVE_WALLET_CLEAR_FILTERS" desc="Clear filters label">Clear filters</message>
   <message name="IDS_BRAVE_WALLET_SHOW_MORE" desc="Show more label">Show more</message>
   <message name="IDS_BRAVE_WALLET_DETAILS" desc="Details label">Details</message>
+  <message name="IDS_BRAVE_WALLET_SWITCH_TO_SHIELDED_ACCOUNT" desc="Switch to shielded account button text">Switch to shielded account</message>
```

Comment:
Should it be "Switch to **a** shielded account"?

Reply:
Updated!

![Screenshot 39](https://github.com/user-attachments/assets/5da4ab11-0d00-403b-af52-6bf44ccea54f)